Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:12-alpine as base
+WORKDIR /build
+COPY kafmesh-ui/package.json .
+COPY kafmesh-ui/yarn.lock .
+RUN yarn
+COPY ./kafmesh-ui/ ./
+
+FROM base AS build-kafmesh-ui
```

Comment:
in other projects, we have base / multiple builds / final because we do something like:
1. install dependencies on base
2. create multiple build targets.  copy only the source files needed for each image over to them and build independently.
3. create multiple final/release targets, one for each artifact output from 2

no harm in keeping it here, but since we're only building one thing there's no real reason to have an additional build target defined

Reply:
I am thinking that storybook might be an essential tool in getting this up and running, which might constitute a structure like this, no? Though, of course, I don't really have a full understanding of how storybook might be deployed in this circumstance.

Reply:
Oh, good point.  I guess I was picturing Storybook only running locally.  But we'd definitely want to have components up for review beforehand.  Good call.